src/domain/innovation-space/branding/branding.entity.ts

```diff
@@ -1,13 +1,8 @@
-import { Column, Entity, JoinColumn, OneToOne } from 'typeorm';
+import { Column, Entity } from 'typeorm';
 import { BaseAlkemioEntity } from '@domain/common/entity/base-entity';
-import { Visual } from '@domain/common/visual';
 
 @Entity()
 export class Branding extends BaseAlkemioEntity {
-  @OneToOne(() => Visual)
```

Comment:
Removing this is not part of the scope of this PR and requires further discussion.